setup.py

```diff
@@ -43,6 +43,12 @@ def get_long_description():
     author="A lot of people",
     author_email="code-quality@python.org",
     url="https://github.com/PyCQA/pyflakes",
+    project_urls={
+        "Source": "https://github.com/PyCQA/pyflakes",
+        "Changelog": "https://github.com/PyCQA/pyflakes/blob/master/NEWS.rst",
+        "Bug Tracker": "https://github.com/PyCQA/pyflakes/issues",
```

Comment:
this is also already linked because we used github as the `url` so we don't need this either